tingbot/graphics.py

```diff
@@ -410,7 +395,7 @@ def _get_frames(self, pil_image):
             else:
                 palette = base_palette
 
-            pygame_image = pygame.image.fromstring(pil_image.tostring(), pil_image.size, pil_image.mode)
+            pygame_image = pygame.image.fromstring(pil_image.tobytes(), pil_image.size, pil_image.mode)
```

Comment:
So I tried to revert it back... and got 
```
Exception: tostring() has been removed. Please call tobytes() instead.
```
Sigh. I'll put in  a try:except: block to cope with both situations. I've added a test so that any problems get picked up